runner.py

```diff
@@ -274,6 +274,32 @@ def process_data(with_indels, dust, evalue, fastq_files, log, max_basecall_itera
     return reference_file
 
 
+def validate_input(output_dir, input_dir, reference_file):
+    if os.path.exists(output_dir):
+        list_dir = os.listdir(output_dir)
+        if len(list_dir) > 0:
+            if not ((len(list_dir) == 1) and (list_dir[0] == 'pbs_logs')):
+                raise Exception("output_dir must be path to a new or empty directory!")
+    if not os.path.isfile(reference_file):
+        raise Exception("Reference_file must exist!")
+    else:
+        if ".fasta" not in os.path.basename(reference_file):
+            raise Exception("Reference_file must end with .fasta!")
+    if not os.path.isdir(input_dir):
+        raise Exception("Input_dir must exist!")
+    list_dir = os.listdir(input_dir)
+    if len(list_dir) > 0:
```

Comment:
there is actually a utility functions that lists files by extension which you can use for this.
it's in utils.py

Reply:
fixed- used the function for both cases